go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
 	github.com/gardener/etcd-druid v0.3.0
-	github.com/gardener/gardener v1.15.1-0.20210118135118-1c2defb1ce0b
+	github.com/gardener/gardener v1.15.1-0.20210127160255-46acdf6fc0ba
```

Comment:
```suggestion
	github.com/gardener/gardener v1.16.0
```

Reply:
Made the suggested change [here](https://github.com/gardener/gardener-extension-provider-aws/pull/262/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R11) and ran `make revendor`.